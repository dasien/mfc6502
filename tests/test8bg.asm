JSR Init
STX $01

Init: INX
INX
INX
RTS
