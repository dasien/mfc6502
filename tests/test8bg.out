20 05 01
86 01
E8 
E8 
E8 
60 
